srcpkgs/ft2-clone/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'ft2-clone'
+pkgname=ft2-clone
+version=1.10
+revision=1
+archs="i686 x86_64"
+makedepends="alsa-lib-devel SDL2-devel"
+short_desc="Modern clone of the FastTracker II music tracker"
+maintainer="rc-05 <rc23@email.it>"
+license="BSD-3-Clause"
+homepage="https://16-bits.org/ft2.php"
+distfiles="https://github.com/8bitbubsy/${pkgname}/archive/v${version}.tar.gz"
+checksum=0414bc9d78d83306ff6423ef8f63be7f2efcbdae6b71e26f379b038554ccf611
+
+pre_build() {
+	chmod +x make-linux.sh
+}
+
+do_build() {
+	./make-linux.sh
```

Comment:
This script hardcodes gcc which breaks cross-compiling and uses `-march=native -mtune=native` which shouldn't be set for packaging.
There is a CMakeLists.txt, so maybe give `build_style=cmake` a try.

Reply:
I've changed the template file accordingly and now I'm cross-compiling the package locally to see if cross-compilation can be achieved or not. Going to amend the commit as soon as it's finished.